react/testing_5/src/app/screens/Dashboard/components/SignupForm/index.test.js

```diff
@@ -0,0 +1,135 @@
+import Faker from 'faker';
+import Puppeteer from 'puppeteer';
+
+import { TEST_BASE_URL } from '../../../../../config/test';
+
+const person = {
+  firstName: Faker.name.firstName(),
+  lastName: Faker.name.lastName(),
+  email: Faker.internet.email()
+};
+
+describe('Signup Form', () => {
+  let browser;
+  let page;
+
+  beforeAll(async () => {
+    browser = await Puppeteer.launch();
+    page = await browser.newPage();
+
+    page.emulate({
+      viewport: {
+        width: 500,
+        height: 900
+      },
+      userAgent: ''
+    });
+  });
+
+  beforeEach(async () => page.goto(TEST_BASE_URL));
+
+  afterAll(() => browser.close());
+
+  it('Can submit signup form', async () => {
+    await page.waitForSelector('.signup-form');
+    await page.click('input[name=firstName]');
+    await page.type('input[name=firstName]', person.firstName);
+    await page.click('input[name=email]');
+    await page.type('input[name=email]', person.email);
```

Comment:
If the case, we need to specify that we should use `field` names from constants

Reply:
Yes, of course. This is just an example for you, they will be deleted before merging